src/main/java/pl/psnc/dei/util/ImportNameCreatorUtil.java

```diff
@@ -15,4 +15,7 @@ public static String generateImportName(String value) {
         return String.format(TITLE_PATTERN, value, new Date(), new Date());
     }
 
+    public static boolean isMatchingImportTitlePattern(String text) {
+        return text.matches("IMPORT_[A-Za-z]+_[0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+");
```

Comment:
This regex does not match string `IMPORT_1914-1918_2021-07-27T15:56:11`, which is correct string for one of datasets.

Reply:
Why do we even need a method like this? There is no requirement for analyzing import name. Only the automatically generated import name should match the pattern defined in TITLE_PATTERN and after creation there is no need to verify that. User generated names have no limits.

Reply:
> This regex does not match string `IMPORT_1914-1918_2021-07-27T15:56:11`, which is correct string for one of datasets.

Relaxed regex

Reply:
more information, why to use regex than dirtiness checking to be found in code